contracts/libraries/wrappers/IterableAppendOnlySetWrapper.sol

```diff
@@ -23,15 +23,11 @@ contract IterableAppendOnlySetWrapper {
         return data.last;
     }
 
-    function size() public view returns (uint256) {
-        return data.size();
+    function size() public view returns (uint96) {
+        return data.size;
     }
 
     function next(address value) public view returns (address) {
         return data.next(value);
     }
-
-    function atIndex(uint256 index) public view returns (address) {
```

Comment:
I don’t see how removing this is relevant here. This is a standard implement of getting and element by index from such a data structure and I am under the impression. That this is used somewhere. 

However, I  understand the implementation is    subject to the same issue and suppose it would be overall better for someone to fetch the entire set and get their elements as needed off chain.

Reply:
We are not currently using it and my worry was that the fact this is using an unbounded loop is hidden behind a layer of abstraction making it easy for people to accidentally use it in code that would break if the set got too big. I can put it back if you feel it's valueable to keep.

Reply:
Nah. There is no need to keep it if it isn't used.

Reply:
Thinking about this a but more, wouldn't we avoid a lot of mess if `size` was just stored in the contract?

Reply:
Technically we store the size in the contract (since the contract stores the self storage struct). The point for this library is to abstract set operations such as insertion and iteration (and arguably keeping track of size).